modules/rubiconBidAdapter.js

```diff
@@ -185,7 +185,17 @@ export const spec = {
         }
       }
 
-      const bidFloor = parseFloat(utils.deepAccess(bidRequest, 'params.floor'));
+      let bidFloor;
+      if (typeof bidRequest.getFloor === 'function' && !config.getConfig('rubicon.disableFloors')) {
+        let floorInfo = bidRequest.getFloor({
+          currency: 'USD',
+          mediaType: 'video',
+          size: parseSizes(bidRequest, 'video')
+        });
+        bidFloor = typeof floorInfo === 'object' && floorInfo.currency === 'USD' && !isNaN(floorInfo.floor) ? parseFloat(floorInfo.floor) : undefined;
```

Comment:
I'd suggest using `!isNaN(parseInt(floorInfo.floor))`
This protects against some odd behaviors with `isNaN`. For example, `isNaN(null)` and `isNaN('')` both return false. `parseFloat` of null or empty string would then return NaN.

Same suggestion for line 507

Reply:
Nice thanks. Will update